source/kit/components/ui/inlineText.js

```diff
@@ -0,0 +1,143 @@
+(function(undefined) {
+    'use strict';
+
+    angular
+        .module('bmg.components.ui')
+        .directive('inlineText', inlineText);
```

Comment:
On a general note: There's quite a bit of code duplication present within all the inline*.js files, apparently. Can this be modified via inheritance/prototyping/abstraction, resulting in an inlineBase.js directive and several inline*.js directives all inheriting the same(-ish) behavior?